doc/api_changes.rst

```diff
@@ -181,7 +181,7 @@ DIPY 0.14 Changes
 ``dipy.io.trackvis`` module is deprecated. Use ``dipy.io.streamline`` instead. Furthermore,
 ``load_trk`` and ``save_trk`` from ``dipy.io.streamline`` is highly recommended for managing streamlines.
 When you create streamlines, you should use ``from dipy.tracking.streamlines import Streamlines``. This new
-object uses much less memory and it is easier to process.
+the object uses much less memory and it is easier to process.
```

Comment:
Please revert this change:

```suggestion
object uses much less memory and it is easier to process.
```

Reply:
Thanks for pointing out.
Even I don't remember why did I put 'the' here when it made no sense